cumulus/thin-egress.tf

```diff
@@ -23,7 +23,7 @@ module "thin_egress_app" {
   urs_auth_creds_secret_name         = aws_secretsmanager_secret.thin_egress_urs_creds.name
   use_cors                           = var.use_cors
   vpc_subnet_ids                     = data.aws_subnets.subnet_ids.ids
-  tags                               = local.default_tags
+  tags                               = merge(local.default_tags, {DAR = "YES"})
```

Comment:
Does TEA create a bucket?  I'm not sure why this is needed.

Reply:
Yes, the TEA terraform module creates a bucket for the lambda source code zip, dependency layer zip, and CloudFormation.

We are in the process of releasing a new version of TEA that adds the DAR tag to the bucket: https://github.com/asfadmin/thin-egress-app/releases/tag/tea-release.2.0.1

I think it would be better to update the version of TEA rather than pass the tags in like this, because doing it this way will also tag the CloudFormation stack with the DAR tag meaning it will tag every resource (lambdas, queues, API gateways, etc) with the DAR tag.

Reply:
Good to know - agreed, it is better to update in TEA than to include this tag with everything in the TEA stack.  I'll remove this from the PR given that this work is ongoing.